### PR TITLE
Included tools to the pip package

### PR DESCRIPTION
**Pull request by jesper-friis**
Closes #472

If installed in an environment, the user will most likely have to add LD_LIBRARY_PATH to the activate script.

### CODE REVIEW

File: python/setup.py [hunk 1]
```diff
@@ -208,12 +208,18 @@ def build_extension(self, ext: CMakeExtension) -> None:
     # See issue #222: https://github.com/SINTEF/dlite/issues/222
     # extras_require={"all": extra_requirements},
     packages=["dlite"],
+    scripts=[
+        #str(SOURCE_DIR / "bindings" / "python" / "scripts" / "dlite-validate"),
+    ],
```
Comment (CasperWA) [inline python/setup.py:213]:
```suggestion
```
If it's not used, it shouldn't be included.

If what you wish to do is to have callable scripts upon installation, use the `console_scripts` key in the `entry points` dictionary ([setuptools documentation](https://setuptools.pypa.io/en/latest/userguide/entry_point.html)).

File: python/setup.py [hunk 2]
```diff
@@ -208,12 +208,18 @@ def build_extension(self, ext: CMakeExtension) -> None:
...
     package_data={
         "dlite": [
             dlite_compiled_ext,
             dlite_compiled_dll_suffix,
             str(Path(".") / "share" / "dlite" / "storage-plugins" /
                 dlite_compiled_dll_suffix),
+            str(Path(".") / "bin" / "dlite-getuuid"),
+            str(Path(".") / "bin" / "dlite-codegen"),
+            str(Path(".") / "bin" / "dlite-env"),
```
Comment (CasperWA) [inline python/setup.py:222]: Could you not just do a single line with a glob expression?
E.g.,: `str(Path(".").resolve() / "bin" / "*")` or something similar.

**Review by CasperWA — CHANGES_REQUESTED** [review]

File: python/setup.py (hunk 1 above)
Comment (jesper-friis) [reply python/setup.py:213]: It awaits PR #461 to be merged. When that is done, it should be uncommented.

File: python/setup.py (hunk 2 above)
Comment (jesper-friis) [reply python/setup.py:222]: No, that is dangerous and might pick up things we don't want to install.

**Comment by CasperWA** [comment]
Have you tested this works as intended upon building and installing the Python package?

**Comment by jesper-friis** [comment]
> Have you tested this works as intended upon building and installing the Python package?

How do we get access to the python package before it is made available on pypi?

**Comment by CasperWA** [comment]
> > Have you tested this works as intended upon building and installing the Python package?
> 
> How do we get access to the python package before it is made available on pypi?

I would just build it locally, and possibly create a new virtualenv to install the built wheel?

**Pull request merged**